forui/test/src/widgets/accordion/accordion_controller_test.dart

```diff
@@ -0,0 +1,363 @@
+import 'package:flutter/animation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'package:forui/forui.dart';
+import 'accordion_controller_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<AnimationController>()])
+@GenerateNiceMocks([MockSpec<Animation>()])
+void _setup(List<AnimationController> animationControllers, List<Animation<int>> animations, int length) {
+  animations.clear();
+  animationControllers.clear();
```

Comment:
Why do we need to clear these? Would it be better to return a list of animation controllers and animations as a tuple instead?